workflows/default-workflow.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
rename to meaningful names or delete if not needed

Reply:
fixed in d36b973